documentation/docs/en/30_Uploading-Pictures.md

```diff
@@ -1,18 +1,21 @@
 # Uploading Photos
 
-As of writing this documentation, the photo upload feature is already operational in the test environment we are currently using. However, there might be some changes you'll notice when we go 'live.'
-
 The uploading process is designed to be as simple as possible, and here is how it works:
 
+## via the website
+
 - Go to the website and log in with the user you created earlier.
 - Navigate to the upload screen where a window appears. You can either navigate to a folder containing your photos and select them, or you can go to the folder first, select all the necessary photos, and drag them onto the screen. Once you release them, the uploading process begins.
 
-During the upload, you may need to be patient (at least at the time of writing). Even though there are prompts to go to 'My Photos' in a few places, avoid doing that as it may interrupt the upload.
+Now it is a matter of waiting till the upload is done. You can see the progress for each selected photo, and in between, you can drag additional photos onto the screen, and they will be uploaded as well.
+If the upload is ready, it will say 100%. If something went wrong on a photo, it might say something like 98%. In that case you can scroll down to the picture that failed (shown in red) and press the re-try-button.
 
-The photos you upload are automatically resized to a format that is large enough to be displayed unchanged on the screen but requires much less storage capacity. You can see the progress for each selected photo, and in between, you can drag additional photos onto the screen, and they will be uploaded as well.
+The photos you upload are automatically resized to a format that is large enough to be displayed unchanged on the screen but requires much less storage capacity.
```

Comment:
because it is not just 'less capacity' but 'de overtreffende trap daarvan'.
Not sure how you would phrase that, 'way less'? 'considerable less'?
Yah, i guess the last one, i will change that.